.gitignore

```diff
@@ -4,4 +4,6 @@ book/
 target
 tmp
 .vscode/
-.idea/
\ No newline at end of file
+.idea/
+*.code-workspace
+mdbook-admonish.css
```

Comment:
Why is this needed? The `css` should not be in the root.

Also, what is `*.code-workspace`?
VS Code's directory is typically `.vscode`, no?

Reply:
That keeps happening because mdbook refuses to serve without running `mdbook-admonish install`. To check my edits I have to cause issues to the toml and then clear it out to push. Must have missed that.

as for `*.code-workspace`, that's the file type for defining workspace settings overrides. I use it to force wrap at 150 characters, so I can easily see where to make line breaks.